test/e2e/shared/aws_helpers.go

```diff
@@ -196,7 +196,7 @@ func CheckClassicElbHealthCheck(input CheckClassicElbHealthCheckInput, intervals
 		}
 
 		if *lb.HealthCheck.Target != input.ExpectedTarget {
-			return fmt.Errorf("health check target does not match expected target")
+			return fmt.Errorf("health check target %s does not match expected target %s", *lb.HealthCheck.Target, input.ExpectedTarget)
```

Comment:
Minor nit: `%q` might be easier to read, since it puts quotes around the interpolated string.

Reply:
Changed